packages/storybook/index.html

```diff
@@ -7,13 +7,29 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <title>Component Test Page</title>
   <script src="/src/index.ts" type="module"></script>
+  <!--
+    the following file is public/global.css, it will not have postcss processing
```

Comment:
Determined this experimentally. Even if referenced in `src/index.ts` it did not appear to be getting processed during a `yarn watch:vite`. If referenced by `src/index.ts` `yarn build` results in the contents of `public/global.css` being processed by postcss and put in `dist/style.css` but `dist/global.css` was not processed and just copied as is.